widget/diagramwidget/diagram.go

```diff
@@ -38,6 +37,9 @@ type DiagramWidget struct {
 	// DesiredSize specifies the size of the displayed diagram. Defaults to 800 x 600
 	DesiredSize fyne.Size
 
+	// MinimumSize is the minimum size for the diagram widget
+	MinimumSize fyne.Size
```

Comment:
I don't think this should be exposed as a field - the function `MinSize()` already returns that information.